### PR TITLE
DEVX-318: add pre-configured connector configurations for CP quickstart

### PR DESCRIPTION
**Pull request by ybyzek**

### CODE REVIEW

File: connector_pageviews.config [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "datagen-pageviews",
+  "config": {
+    "connector.class": "io.confluent.kafka.connect.datagen.DatagenConnector",
+    "kafka.topic": "pageviews",
+    "quickstart": "pageviews",
+    "key.converter": "org.apache.kafka.connect.storage.StringConverter",
+    "value.converter": "org.apache.kafka.connect.json.JsonConverter",
```
Comment (C0urante) [inline connector_pageviews.config]: Unless the connector requires these converters in order to work (and I don't think it does), we can probably leave the `key.converter` and `value.converter` configs out of these sample configs.

Comment (ybyzek) [reply connector_pageviews.config]: We need them for Confluent CLI to override the default which is the AvroConverter

Comment (C0urante) [reply connector_pageviews.config]: Does the data generated by the connector not work with the `AvroConverter`?

Comment (ybyzek) [reply connector_pageviews.config]: The connector does work with Avro, but the quickstarts are currently not using Avro

Comment (C0urante) [reply connector_pageviews.config]: Gotcha, thanks for clarifying.

**Pull request merged**